app/src/main/res/values/strings.xml

```diff
@@ -704,6 +705,7 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_recycling_type_plastic_packaging">Plastic packaging only</string>
     <string name="quest_recycling_type_scrap_metal">Scrap metal</string>
     <string name="quest_recycling_type_any_glass">Any glass</string>
-    <string name="quest_recycling_glass_title">Can only glass bottles be recycled here, or any type of glass?</string>
+    <string name="quest_recycling_glass_title">Can only glass bottles and jars be recycled here, or any type of glass?</string>
+    <string name="quest_determineRecyclingGlass_description_any_glass">\"any glass\" includes window glass, glassware, tempered glass, optical glass, light bulb glass, mirrors and other that are often not accepted.</string>
```

Comment:
Sounds a bit odd to enumerate what "any glass" encompasses. Any glass is any glass. How about giving only a few well known examples of glass that are definitely not glass bottles or similar.

```suggestion
    <string name="quest_determineRecyclingGlass_description_any_glass">Examples of glass that is often not accepted is such glass as from window panes, glassware, light bulbs or mirrors.</string>
```